rotkehlchen/chain/ethereum/modules/convex/convex_cache.py

```diff
@@ -108,6 +108,7 @@ def query_convex_data(
         inquirer: 'EthereumInquirer',
         cache_type: Literal[CacheType.CONVEX_POOL_ADDRESS],
         msg_aggregator: 'MessagesAggregator',  # pylint: disable=unused-argument  # argument is unused to keep the interface consistent with the other functions because they all are called similarly in ensure_cache_data_is_updated and refresh_general_cache
+        reload_all: bool,  # pylint: disable=unused-argument
```

Comment:
i don't think this is useful here since partial update isn't implemented for this protocol. similarly for gearbox, extrafi & balancer.